lredis/cqueues.lua

```diff
@@ -11,7 +11,14 @@ local mt = {
 	__index = methods;
 }
 
+-- override the default socket handler so that it returns all errors rather than
+-- throwing them.
+local function socket_error_handler(socket, method, code, level)
+	return code, level
```

Comment:
might need more than this, e.g.
  - turning it into a string
  - prettier TLS errors
  - handling timeouts

Reply:
This is the same interface used by the ```cqueues``` default handler.  The only difference is that this does ```return code, level``` for all errors; ```cqueues``` does this for ```ETIMEDOUT``` and ```EPIPE``` but ```assert()```'s everything else, which again is what we're trying to avoid.